content/english/microbit/_index.md

```diff
@@ -10,9 +10,7 @@ icon: "fas fa-microchip"
 
 ### An introduction to JavaScript and hardware using [MicroBits](https://microbit.org/guide/)
 
-[logo]: https://media.giphy.com/media/1nOL7s74KmSk0zDlDD/giphy.gif
-
-![alt text][logo]
+<p style="text-align: center; "><img src="https://media.giphy.com/media/1nOL7s74KmSk0zDlDD/giphy.gif" alt="Nuvi logo" width="350"/></p>
```

Comment:
Can `![]()` not be used here?

Reply:
Nope, because we needed centered and not that big.

Reply:
I tried but look too big so changed it back.